src/sendHappyMessages.js

```diff
@@ -0,0 +1,30 @@
+import { room } from "./room";
+import { randomInt } from "./utils";
+
+
+const happyMessagesList = [
+  "JUGAS MEJOR QUE MANANA",
+  "RETIRATE HIJO DE PUTA",
+  "AVERIGUASTE DE QUE JUGAS",
+  "TENES LAG EN EL CEREBRO",
+  "DOWN",
+  "ESCUPI EL FITITO",
+  "HACE ALGO",
+  "CORRETE QUE ESTAN JUGANDO",
+  "TE VA A ORINAR UN PERRO"
+]
+
+const happyMessages = () => {
+  var players = room.getPlayerList();
+  var playerList = players.filter((player) => player.position !== null);
+  if (players.length > 1 && playerList.length > 0) {
```

Comment:
Necesitas el otro condicional?
```suggestion
  if (playerList.length > 0) {
```